run.py

```diff
@@ -0,0 +1,43 @@
+from ocdata.adsorptions import sample_structures
+from ocdata.vasp import run_vasp, write_vasp_input_files
+import numpy as np
+import time
+import sys
+import pickle
+
+val = sys.argv[1]
+
+np.random.seed(int(val))
+
+BULK_PATH = "/checkpoint/sidgoyal/electro_may6_100k_round2/bulk/"
+ADSORBED_BULK_PATH = "/checkpoint/sidgoyal/electro_may6_100k_round2/bulkadsorbate/"
+
+def write_pkl(d, path):
+    with open(path, 'wb') as f:
+        pickle.dump(d, f)
+
+s = time.time()
+success = True
+try:
+    adsorbed_bulk_dict, bulk_dict = sample_structures()  
```

Comment:
this is still using the old `sample_strcture` implementation right?

Reply:
Yes, run.py calls methods from adsorptions.py. I didn't make any changes to either of these files (except for some minor edits), I'm mostly leaving them here for now for reference, i.e. to compare the code in my bulk/surface/combined objects to the methods in adsorptions.py

Reply:
Cool.